packages/template-retail-react-app/config/default.js

```diff
@@ -23,6 +23,9 @@ module.exports = {
             social: {
                 enabled: false,
                 idps: ['google', 'apple']
+            },
+            resetPassword: {
+                callbackURI: '/reset-password-callback'
             }
```

Comment:
To enable customization of the reset password callback path, I am adding a key here under `login` in the config file. When we build the SSR POST endpoint we then extract and use this value, and if it is not set, then it defaults to `/reset-password-callback`.